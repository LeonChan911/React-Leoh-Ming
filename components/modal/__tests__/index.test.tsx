import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Modal from '../index';

describe('Modal', () => {
  test('should render default', () => {
    const { container } = render(<Modal />);
    expect(container).toMatchSnapshot();
  });
});
