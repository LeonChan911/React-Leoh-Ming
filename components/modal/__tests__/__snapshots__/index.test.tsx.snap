// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render default 1`] = `
<div>
  <div
    class="lm-modal-mask lm-modal-mask-animation lm-modal-mask-hidden"
  >
    <div
      class="lm-modal-content lm-modal-content-animation lm-modal-content-hidden"
    />
  </div>
</div>
`;
